### ISSUE TITLE
[BUG] object detection is not available on 0.7

### ISSUE THREAD

**Issue by sungeuns** [autogluon/autogluon#3082]
- [x] I have checked that this bug exists on the latest stable version of AutoGluon
- [ ] and/or I have checked that this bug exists on the latest mainline of AutoGluon via source installation

**Describe the bug**

When I install autogluon using `pip install autogluon`, I can't use object detection from `MultiModalPredictor` because error below happened:

<img width="1275" alt="image" src="https://user-images.githubusercontent.com/1274149/227987424-6446ec2d-b922-4abe-be25-19d72429c3ea.png">

When I trying to find that file in site-package install path, it really not exist. (But in the source code it is exist)

But if I install it from the source code like below, it works well.
```
git clone https://github.com/autogluon/autogluon
cd autogluon && ./full_install.sh
```

The 0.7 document not explained about this issue, so I think doc need to fixed for object detection function.

**Expected behavior**

Latest AutoGluon should have object detection function.

**To Reproduce**

- In Jupyter notebook (SageMaker studio, notebook, ...) it can be reproduced.
- Also when I use 0.7 version from AWS DLC ([link](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#autogluon-training-containers)) same issue also happened.

**Comment by DaBaiTuu** [comment]
same problem,but when i copy the file into the specific path,there is still a error:

"/home/dongsongb/anaconda3/envs/AutoML/lib/python3.8/site-packages/torch/utils/d
ata/_utils/collate.py", line 151, in collate
    raise TypeError(default_collate_err_msg_format.format(elem_type))
TypeError: default_collate: batch must contain tensors, numpy arrays, numbers, 
dicts or lists; found <class 'NoneType'>


And my packages are exactly same as the demo.

**Comment by zhiqiangdon** [comment]
@FANGAreNotGnu Can you take a look at this issue?

**Comment by zhiqiangdon** [comment]
Should be fixed by #3158. You can try our nightly build version to verify this.

**Comment by zhiqiangdon** [comment]
Try `pip install autogluon --pre`.
